### ISSUE TITLE
Upgrade @hpcc-js/wasm to 1.01

### ISSUE THREAD

**Issue by tuckergordon** [magjac/d3-graphviz#181]
As of writing this, @hpcc-js/wasm is on version 1.01. Would love to see package.json updated to reflect this.

A little story as to why this would be nice. I'm hoping that by writing this, others may find there way here and avoid the issues that I had.

I'm trying to use d3-graphviz with Svelte. Following the workaround in  #152, I was able to get it _mostly_ working. However, it only worked if `useWorker` was `true` and `drawNode` and `drawEdge` failed due to undefined `layoutSync`. And even then, I was getting this error:

```
Uncaught (in promise) RuntimeError: abort(LinkError: WebAssembly.instantiate(): memory import 49 has no maximum limit, expected at most 4294967295). Build with -s ASSERTIONS=1 for more info.
```

Then I noticed issue [#30](https://github.com/hpcc-systems/hpcc-js-wasm/issues/30) on [@hpcc-js/wasm](https://github.com/hpcc-systems/hpcc-js-wasm), which includes this note:

> It looks like the "@hpcc-js/wasm" JavaScript bundled into your WebApp is loading a different version of the wasm file than the one it was built with.
> 
> This typically happens if you bumped the version number of @hpcc-js/wasm, but didn't copy the matching version of the wasm file to your web server.

That was the clue I needed. When I downgraded @hpcc-js/wasm to match d3-graphviz, everything worked as expected.

This error only appeared because I'm using a framework (Svelte) and had to manually add the @hpcc-js/wasm files to the build. In cases where people don't have to do that, @hpcc-js/wasm is just a dependency of d3-graphviz so the versions will match. 

TLDR: if you're having trouble and you're using a framework, make sure the @hpcc-js/wasm version that you're using matches the one that d3-graphviz is using.

**Comment by magjac** [comment]
I''m planning to make I new release when [Graphviz 2.46.0](https://gitlab.com/graphviz/graphviz/-/merge_requests/1672) has been released and a new @hpcc-js/wasm release is available based on it.

I don't have any plans to make new d3-graphviz release every time @hpcc-js/wasm makes a new release if there are no significant enhancements or bug fixes that benefit d3-graphviz users in it.

**Comment by magjac** [comment]
Resolved by https://github.com/magjac/d3-graphviz/commit/2b965f57c7b7a9982ad3b6fffa15e7e811d960f3 which was included in [v3.2.0](https://github.com/magjac/d3-graphviz/blob/master/CHANGELOG.md#320).
